docs/integrations/microsoft-azure/azure-database-for-mysql.md

```diff
@@ -16,46 +16,226 @@ The below instructions applies to Azure Database for MySQL with Flexible Server
+   1. Select `AllMetrics`.
+   1. Use the Event hub namespace created by the ARM template in Step 2 above. You can create a new Event hub or use the one created by ARM template. You can use the default policy `RootManageSharedAccessKey` as the policy name. <br/><img src={useBaseUrl('img/send-data/azureflexible-mysqlserver-metrics.png')} alt="Azure flexible mysql server metrics" style={{border: '1px solid gray'}} width="800" />
+
 
 ### Configure logs collection
 
+#### Diagnostic logs
+
 In this section, you will configure a pipeline for shipping diagnostic logs from Azure Monitor to an Event Hub.
 
-1. To enable slow query logs, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/mysql/flexible-server/concepts-slow-query-logs#configure-slow-query-logging).
-2. To enable audit logs, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/mysql/flexible-server/tutorial-configure-audit#configure-auditing-by-using-the-azure-portal).
-3. To set up the Azure Event Hubs source in Sumo Logic, refer to [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
-4. To create the Diagnostic settings in Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/mysql/flexible-server/tutorial-query-performance-insights#set-up-diagnostics). Perform below steps for each Azure Database for MySQL server that you want to monitor.
-   * Choose `Stream to an event hub` as the destination.
-   * Select `MySqlSlowLogs` and `MySqlAuditLogs`.
-   * Use the Event hub namespace and Event hub name configured in previous step in destination details section. You can use the default policy `RootManageSharedAccessKey` as the policy name.
+1. To set up the Azure Event Hubs source in Sumo Logic, refer to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
+1. To create the diagnostic settings in Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/data-factory/monitor-configure-diagnostics). Perform the steps below for each Azure Redis cache account that you want to monitor.
+   1. Choose **Stream to an event hub** as the destination.
+   1. Select `allLogs`.
+   1. Use the Event Hub namespace and Event Hub name configured in the previous step in the destination details section. You can use the default policy `RootManageSharedAccessKey` as the policy name.
+1. Tag the location field in the source with right location value. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Tag-Location.png')} alt="Azure Redis Cache Tag Location" style={{border: '1px solid gray'}} width="400" /> <br/><img src={useBaseUrl('img/send-data/azureflexible-mysqlserver-logs.png')} alt="Azure flexible mysql server logs" style={{border: '1px solid gray'}} width="800" /> 
+
+#### Activity Logs
```

Comment:
Need to add section for 
Enabling error logs (along with the required query parameters)
Enabling Slow query logs (along with the required query parameters)
Enabling Audit  logs (along with the required query parameters)


Also is there any extra step required for enabling any health related logs and security (cloud defender ) related logs?